### ISSUE TITLE
My generic gamepad only works the Y and B buttons and the joysticks (Tested on Github and Playstore version).

### ISSUE THREAD

**Issue by diog2003** [PojavLauncherTeam/PojavLauncher#6282]
### Describe the bug

When pairing my generic gamepad, only four commands respond: The left and right joysticks, and the Y and B buttons. The gamepad model in question is the Lotus LT CT-003.

### The log file and images/videos

[latestlog.txt](https://github.com/user-attachments/files/17774911/latestlog.txt)

### Steps To Reproduce

```markdown
1. Start PojavLauncher
2. Connect the gamepad
```

### Expected Behavior

I hope there is a solution and that I and many others can play peacefully without major bugs. Thank you.

### Platform

```markdown
- Device model: Redmi Note 8 Pro
- CPU architecture: Arm
- Android version: 9
- PojavLauncher version: foxglove-20241020-ced81e-v3_openjdk
```

**Comment by GaucovyAero** [comment]
check if it works properly with the gamepad tester in the play store first

**Comment by diog2003** [comment]
Hello. I tested the Github and Playstore version on two different smartphones, with two different gamepads, and the same bug happens in the same way on all of them. Both smartphones are also Mediatek, I believe the bug occurs on this type of chipset.

**Comment by GaucovyAero** [comment]
okay, what about the gamepad tester, do they behave correctly in that app?
(if they do, it might be fixable)

**Comment by Jack0ForREAL** [comment]
i have the same issue please fix

im able to map all my buttons (the launcher askes me to press a b c y etc) but im only able to move around ingame
the y and b is mapped to ESC for some reason and yes the controller works perfectly fine in my computer and in my phone (used gamesir app to test)

**Comment by GaucovyAero** [comment]
> i have the same issue please fix
> 
> im able to map all my buttons (the launcher askes me to press a b c y etc) but im only able to move around ingame the y and b is mapped to ESC for some reason and yes the controller works perfectly fine in my computer and in my phone (used gamesir app to test)

use the gamepad tester app first and see if it works there and report back

**Comment by GaucovyAero** [comment]
closing because of inactivity

**Comment by diog2003** [comment]
I also tested with some gamepad mappers, but I was not successful.

**Comment by GaucovyAero** [comment]
yeah thats pretty unfortunate :/

**Comment by diog2003** [comment]
> yeah thats pretty unfortunate :/

But I believe you can solve this problem.

**Comment by GaucovyAero** [comment]
nope, if you cant get it working in other apps, it also cant work in pojav
